Startup.cs

```diff
@@ -47,12 +57,43 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
 
             app.UseWebSockets();
-            app.UseGraphQLWebSockets<SampleSchema>();
+            // Replaced the HttpMiddleware with paths for each schema
+           // app.UseGraphQL<ForecastSchema, GraphQLHttpMiddleware<ForecastSchema>>();
+            app.UseGraphQL<ForecastSchema>("/api/forecast");
+            //app.UseGraphQL<TodoSchema, GraphQLHttpMiddleware<TodoSchema>>();
+            app.UseGraphQL<TodoSchema>("/api/todo");
+
+            app.UseGraphQLWebSockets<ForecastSchema>();
+            app.UseGraphQLWebSockets<TodoSchema>();
 
-            app.UseGraphQL<SampleSchema, GraphQLHttpMiddleware<SampleSchema>>();
+            /*
+             * This will create two endpoints, based on https://github.com/graphql-dotnet/examples/blob/master/src/AspNetCoreMulti/Example/Startup.cs
+             */
+            app.UseGraphQLPlayground(new GraphQLPlaygroundOptions
+            {
+                GraphQLEndPoint = "/api/forecast",
+                Path = "/ui/playground",
+                BetaUpdates = true,
+                RequestCredentials = RequestCredentials.Omit,
+                HideTracingResponse = false,
 
+                EditorCursorShape = EditorCursorShape.Line,
+                EditorTheme = EditorTheme.Dark,
+                EditorFontSize = 14,
+                EditorReuseHeaders = true,
+
+                PrettierPrintWidth = 80,
+                PrettierTabWidth = 2,
+                PrettierUseTabs = true,
+
+                SchemaDisableComments = false,
+                SchemaPollingEnabled = true,
+                SchemaPollingEndpointFilter = "*localhost*",
+                SchemaPollingInterval = 5000
+            });
```

Comment:
I believe we should only have one GraphQL endpoint. The client shouldn't have to make multiple connections to retrieve different data sources.